modules/ui/ProfilingWindow.py

```diff
@@ -0,0 +1,55 @@
+import faulthandler
+
+import customtkinter as ctk
+from scalene import scalene_profiler
+
+from modules.util.ui import components
+
+
+class ProfilingWindow(ctk.CTkToplevel):
+    def __init__(self, parent, *args, **kwargs):
+        ctk.CTkToplevel.__init__(self, parent, *args, **kwargs)
+        self.parent = parent
+
+        self.title("Profiling")
+        self.geometry("512x512")
+        self.resizable(True, True)
+        self.wait_visibility()
+        self.focus_set()
+
+        self.grid_rowconfigure(0, weight=0)
+        self.grid_rowconfigure(1, weight=0)
+        self.grid_rowconfigure(2, weight=1)
+        self.grid_columnconfigure(0, weight=1)
+
+        components.button(self, 0, 0, "Dump stack", self._dump_stack)
+        self._profile_button = components.button(
+            self, 1, 0, "Start Profiling", self._start_profiler)
```

Comment:
Some kind of information (maybe a tooltip) that explains how to use this would be nice. When I press the button without setting the PROFILE env variable, I just get a non descriptive error message in the console.